Modules/mathmodule.c

```diff
@@ -2016,6 +2016,79 @@ math_factorial(PyObject *module, PyObject *arg)
 }
 
 
+/*[clinic input]
+math.lcm
+
+	x as a: object
+	y as b: object
+    /
+least common multiple of x and y
+[clinic start generated code]*/
+
+static PyObject*
+math_lcm(PyObject *module, PyObject *a, PyObject *b)
+{
+    PyObject *g, *m, *f, *ab;
+	
+    a = PyNumber_Index(a);
+	
+    if (a == NULL) {
+        return NULL;
+    }
+	
+    b = PyNumber_Index(b);
+	
+    if (b == NULL) {
+        Py_DECREF(a);
+        return NULL;
+    }
+	
+    g = _PyLong_GCD(a, b);
+    Py_DECREF(a);
+    Py_DECREF(b);
+	
+    if (g == NULL) {
+        Py_DECREF(g);
+        return NULL;
+    }     
+	
+    m = PyNumber_Multiply(a, b);
```

Comment:
You use formula `a * b // gcd(a, b)`.

It is better to use formula `a // gcd(a, b) * b`.